src/styles/block/nav.scss

```diff
@@ -34,10 +17,17 @@
   }
 
   &__link {
-    color: inherit;
+    color: $main-color;
     text-decoration: none;
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    cursor: pointer;
```

Comment:
no need to specify cursor pointer here (pointer is default value for links)

Reply:
done